jlh/JonFirstApp/app/src/main/java/com/e_conomic/jonfirstapp/MainActivity.java

```diff
@@ -60,7 +62,7 @@ private void setupFragments() {
         FragmentManager fragmentManager = getSupportFragmentManager();
 
         // Setup main fragment
-        main_fragment = (MainFragment) fragmentManager.findFragmentById(R.id.fragment_main);
+        mainFragment = (MainFragment) fragmentManager.findFragmentById(R.id.fragment_main);
```

Comment:
This looks nice. At a later point you will see that the `findFragmentById` dance can be avoided, in which case an auto-generated `main_fragment` will be magically vailable.